lib/Transformer.js

```diff
@@ -62,15 +63,34 @@ class Transformer extends Transform {
     const outputValues = this.transformers
       .map(fn => fn(sourceRow, this.info))
 
-    if (!this.filterFn(outputValues)) {
+    const filterFns = Array.isArray(this.filterFn) ? this.filterFn : [this.filterFn]
+
+    let filter = true
+
+    filterFns.forEach(fn => {
+      if (!fn(outputValues)) filter = false
+    })
+
+    if (!filter) {
```

Comment:
We can collapse this down using Array.every 
```
if (!filterFns.every(f => f(outputValues))) {
  this.filterRowCount()
  return callback(null, null)
}
```

Reply:
I was trying to find something like this! Thanks